inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -84,6 +80,13 @@ public function cron_clean_rows() {
 			return;
 		}
 
+		$old_used_css_ids = $this->database->get_old_used_css();
+		foreach ( $old_used_css_ids as $old_used_css ) {
+			$used_css_item = new UsedCSS_Row( $old_used_css );
```

Comment:
We can use the dependency injection to instantiate this class. We can then use the `init` method to initialize the row instance there at `\WP_Rocket\Engine\Optimization\RUCSS\Admin\Database::get_old_used_css` so this method should return an array of rows to be deleted, This will make tests easier and also separate the concerns.

what do u think?

Reply:
sorry, init method is private so we can't do anything here, the current implementation is the best available solutions.